sdf/1.7/material.sdf

```diff
@@ -25,6 +25,10 @@
     </element>
   </element>
 
+  <element name="render_order" type="double" default="0.0" required="0">
```

Comment:
I don't see any other elements using `float`, but I think it should be possible to use that here and match the C++ implementation. See

https://github.com/osrf/sdformat/blob/fc4dc8c74006074a918f59e81ca1148c38198edc/src/Param.cc#L438